data/avgi/avgi weapons.txt

```diff
@@ -0,0 +1,804 @@
+		"submunition" "Fleck Flack" 30
+
+
+outfit "Fleck Flack"
+	weapon
+		sprite "projectile/fleck"
+			"frame rate" 30
+			"no repeat"
+		"die effect" "missile death"
+		"hit effect" "bombardment impact"
+		"inaccuracy" 18
+			narrow
+		"lifetime" 60
+		"random lifetime" 30
+		"random velocity" -6
+		"shield damage" 6
+		"hull damage" 9
+		"hit force" 90
+		"missile strength" 3
+
+effect "nettle exhaust"
+	sprite "effect/avgi flare/rcs"
+		"random start frame"
+		"scale" 0.5
+	"lifetime" 0
+
+
+outfit `"Orchid" Nuclear Missile`
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 660000
+	thumbnail "outfit/orchid missile"
+	"mass" 0.9
+	"orchid capacity" -1
+	description "Unlike humanity, the Avgi do not have a strong taboo against the use of nuclear weapons in space. To them, nuclear weapons are simply another tool, albeit one with a particularly high energy density, to be used when the occasion calls for it. Nuclear weapons have been the Avgi's trump card against the Aberrant, the one weapon they have capable of reliably breaking their advanced shields and atomizing their armor."
+	description "This particular missile carries a compact thermonuclear warhead, a geometrically complex staged device with a far higher yield density than anything humanity has developed. It seems to be the product of centuries of development of nuclear arms, as opposed to humanity's long abstentation from nuclear weapons research."
+
+
+outfit "Orchid Magazine"
+	category "Ammunition"
+	licenses
+		"Avgi Atomics"
+	cost 1440000
+	thumbnail "outfit/unknown"
+	"mass" 1.2
+	"outfit space" -12
+	"orchid capacity" 12
+	ammo `"Orchid" Nuclear Missile`
+	description "Orchid storage"
+
+
+outfit "Orchid Launcher"
+	category "Secondary Weapons"
+	licenses
+		"Avgi Atomics"
+	cost 1370000
+	thumbnail "outfit/orchid missile launcher"
+	"mass" 4.5
+	"outfit space" -18
+	"weapon capacity" -18
+	"gun ports" -1
+	"orchid capacity" 12
+	weapon
+		sprite "projectile/thunderhead fragment"
+			"frame rate" 1
+			"no repeat"
+		sound "finisher"
+		ammo `"Orchid" Nuclear Missile`
+		icon "icon/finisher"
+		"fire effect" "sidewinder fire"
+		"fire effect" "speckle fire"
+		"inaccuracy" 2.4
+		"velocity" 9
+		"lifetime" 30
+		"reload" 60
+		"firing energy" 6
+		"firing heat" 3
+		"radar tracking" 0.6
+		"infrared tracking" 0.9
+		"missile strength" 36
+		"submunition" "Orchid Active"
+		stream
+	description "Unlike humanity, the Avgi do not have a strong taboo against the use of nuclear weapons in space. To them, nuclear weapons are simply another tool, albeit one with a particularly high energy density, to be used when the occasion calls for it. Nuclear weapons have been the Avgi's trump card against the Aberrant, the one weapon they have capable of reliably breaking their advanced shields and atomizing their armor."
+	description "A standard modular launcher such as this one is designed to be easily installed on light to medium warships."
+
+
+outfit "Orchid Launch Array"
```

Comment:
The above comments also apply to this, since it's using the same missiles.
The comparison between the Launch Array and the Launcher is in a much better spot than it is with the Nettle, though.